tests/e2e/autoscaling/autoscaler.go

```diff
@@ -539,7 +553,7 @@ func waitForStatefulSetComplete(cs clientset.Interface, ns *v1.Namespace, ss *ap
 }
 
 func calculateNewPodCountOnNode(cs clientset.Interface, node *v1.Node) int32 {
-	quantity := node.Status.Capacity[v1.ResourceCPU]
+	quantity := node.Status.Allocatable[v1.ResourceCPU]
```

Comment:
Allocatable refers to the real capacity. With this modification, autoscaler tests will be more stable, which is verified by several local tests.